features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/manager/manager.jag

```diff
@@ -98,4 +98,10 @@ var getAPIStoreObj = function() {
     var store = require('apistore');
     return new store.APIStore();
 };
+
+var getUsageClient = function(){
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea